### ISSUE TITLE
Device 27c6:538c compatibility?

### ISSUE THREAD

**Issue by entodoays** [goodix-fp-linux-dev/goodix-fp-dump#31]
This device worked on Ubuntu with the Dell specific driver.
This is the `lsusb` output:
```
Bus 001 Device 002: ID 27c6:538c Shenzhen Goodix Technology Co.,Ltd. Fingerprint Reader
```

**Comment by mpi3d** [comment]
Please be more precise

**Comment by daemonhorn** [comment]
Update: I found out after posting this that someone else (nikicat) already had a more complete driver for 533c open as #75 (stacked on #72/#73), submitted a couple weeks before I got to this independently -- theirs gets an actual finger-present capture with visible ridge detail, mine only ever got a no-finger calibration frame. Closed my duplicate (#76) in favor of theirs and left some supplementary findings as a comment there.

538c/530c are still untested by either of us (no hardware) -- my original note stands: they share the same firmware/command-set per static analysis of the closed blob, so #75's driver plus the methodology at https://github.com/daemonhorn/goodix-533c-re should get you most of the way there if you own that hardware.
